<!DOCTYPE HTML>

<html>
	<head>
		<title>Venkatesh K</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="/assets/css/main.css" />
		<noscript><link rel="stylesheet" href="/assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">
		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<div class="inner">

							<!-- Logo -->
								<a href="/index.html" class="logo">
									<span class="symbol"><img src="/images/home_logo.svg" alt="" /></span><span class="title">Home</span>
								</a>

							<!-- Nav -->
								<nav>
									<ul>
										<li><a href="#menu">Menu</a></li>
									</ul>
								</nav>

						</div>
					</header>

				<!-- Menu -->
					<nav id="menu">
						<h2>Menu</h2>
						<ul>
							<li><a href="/index.html">Home</a></li>
							<li><a href="/about_me/about_me.html">About Me</a></li>
							<li><a href="/research/research.html">Research</a></li>
							<li><a href="/about_me/Venkatesh_CV.pdf">Curriculum Vittae</a></li>
							<li><a href="/other_works/other_works.html">Other Works</a></li>
						</ul>
					</nav>

				<!-- Main -->
					<div id="main">
						<div class="inner">
							<!-- Research experience starts -->
							<h1>Research</h1>
							
							<h2>Analysis of human genomes with genome graphs</h2>
							<p>
								The reference genome is a cardinal element in any genome study. It is a blueprint against which newly sequenced genomes get compared for read mapping, variant calling, and further analyses. The current widely used human reference genome (hg38) represents each chromosome by a linear, continuous string of nucleotide bases. hg38 in its linear form cannot capture the genetic information from different human populations and lead to reference allele bias. <br>
								The study focuses on constructing and using genome graphs as an alternate reference structure for genome research. Scalable computational pipelines were developed by integrating multiple bioinformatics tools to analyze human WGS with pan-genomic graphs. Novel methods of annotating the genome graph were designed to identify functionally significant regions. Detailed analyses were performed to study the structural complexities of the genome graphs.
							</p>
							<!--  -->

							<h2>GenomeIndia: Unveiling unique variants in Indian (sub)populations</h2>
							<p>Previous works have established that substantial differences are present in genomes from different human populations across the world. The Indian population is highly heterogeneous, with an estimate of over 4,500 genetically distinct sub-populations. Moreover, it is also divergent from the much-studied Caucasian genome. The GenomeIndia project aims to sequence the genomes of 10K healthy individuals from the countryâ€™s diverse populace and study them to catalog the common and low-frequency variants in Indians. <br>
							Genome graphs have the potential to detect novel variants in under-represented populations as they overcome reference allele bias. Our study focuses on constructing Indian population-specific genome graphs and using them as reference structures to analyze Indian genomes. The dynamic nature of the genome graphs enhances the detection of pathogenic variants in the Indian population. Functional annotation of Indian genome graphs helps understand the consequences of genetic variations within and between Indian (sub)populations. 
							</p>
							<!--  -->

							<h2>Approachability of genomics and visualization of variants</h2>
							<p>Genomics is a rapidly evolving field that has profoundly impacted biomedical and public health research. As the applications of genomics burgeon into various domains, it calls for tools that empower clinicians and researchers to work with genomic data formats irrespective of their programming expertise. <br>
								We designed and developed SCI-VCF as a cross-platform application to summarise and compare the genetic variants from VCF files. Users can also create customizable interactive visualizations based on the contents of VCF files. The guided GUI setting of the tool empowers researchers and clinicians to perform exploratory genomic data analysis on VCF files irrespective of their programming expertise. The user-friendly and intuitive design of SCI-VCF increases the approachability of genomics to newcomers and introduces genomic data analysis expeditiously.
							</p>
							<!--  -->

							<h2>Polygenic Risk Scores for Common Complex Diseases</h2>
							<p>Complex diseases occur due to the combined effect of multiple genomic mutations at different loci, in conjunction with physical and environmental factors. Genome-wide association Studies (GWAS) statistically assess the strength of the relationship between genomic variants and complex diseases. Using the effect scores from GWASs, polygenic risk scores (PRS) quantify the risk for an individual for common complex diseases by calculating the cumulative effects of the genomic variants in an individual. PRS is most informative for early detection and is valuable when devising intervention and treatment strategies. <br>
								In our study, we built a computational pipeline to calculate PRS for type-2 diabetes for individuals in an Indian cohort. Different GWAS studies were experimented with as base datasets. Various ways to improve the predictive accuracy of the calculated PRS, like linkage disequilibrium treatments, clumping, and p-value thresholding, were explored. Machine learning techniques were employed to enhance the performance of the computational pipeline. This analysis lays the foundation for future studies to develop multi-ethnic and pathway-specific polygenic risk score predictions.
							</p>
							<!--  -->

							<h2>Understanding the Pathogenicity through the Lens of Genome Graphs</h2>
							<p>Understanding the genomic hotspots that drive the transition from asymptomatic carriage to invasive infection remains a challenge in infectious disease research. Traditional reliance on linear reference genomes limits genetic diversity representation and hinders in-depth research.<br>
								In our study, we used genome graphs to investigate the genomic basis of pathogenicity in Streptococcus pneumoniae, a globally relevant bacterium with diverse lineages varying in pathogenicity. We analysed these genome graphs using GViNC, a comprehensive framework for genome graph analyses. We conducted locus-specific genome analyses and applied graph distance metrics to identify key regions of variability differentiating infection from carriage isolates. Our approach offers a scalable framework for integrating genome graphs into routine microbial surveillance, aiding in the study of pathogen evolution in diverse environments
							</p>
							<hr>

							<!-- Tile Section Starts -->
							<h2><a id="sections">More Information</a></h2>
							<p>Click on the respective tiles below to view a catalog of my publications, presentations, and training.</p>
							
							<section class="tiles">
								<article class="style4">
									<span class="image">
										<img src="/images/pic02.jpg" alt="" />
									</span>
									<a href="/research/publications.html">
										<h2>Publications</h2>
										<div class="content">
											<p>List of Publications.</p>
										</div>
									</a>
								</article>
								<article class="style5">
									<span class="image">
										<img src="/images/pic02.jpg" alt="" />
									</span>
									<a href="/research/presentations.html">
										<h2>Presentations</h2>
										<div class="content">
											<p>List of poster presentations and talks given.</p>
										</div>
									</a>
								</article>
								<article class="style3">
									<span class="image">
										<img src="/images/pic03.jpg" alt="" />
									</span>
									<a href="/research/training.html">
										<h2>Conferences/ Training</h2>
										<div class="content">
											<p>List of conferences attended and training undergone.</p>
										</div>
									</a>
								</article>
								
							</section>



							
							
						</div>
					</div>

				<!-- Footer -->
					<footer id="footer">
						<div class="inner">
							<section>
								<h2>Get in touch</h2>
								<ul class="icons">
									<li><a href="mailto:vengatesh.vengatesh89@gmail.com" class="icon solid style2 fa-envelope" target="_blank"><span class="label">Email</span></a></li>
									<li><a href="https://orcid.org/0009-0008-7309-9810" class="icon brands style2 fa-orcid" target="_blank"><span class="label">ORCid</span></a></li>
									<li><a href="https://www.linkedin.com/in/venkatesh-k-0a7045162/" class="icon brands style2 fa-linkedin" target="_blank"><span class="label">LinkedIn</span></a></li>
									<li><a href="https://github.com/venkatk89" class="icon brands style2 fa-github" target="_blank"><span class="label">GitHub</span></a></li>
									<li><a href="https://www.youtube.com/channel/UC-JUCYTcaQftTDMriao5mKg" class="icon brands style2 fa-youtube" target="_blank"><span class="label">YouTube</span></a></li>
								</ul>
							</section>
							<ul class="copyright">
								<li>&copy; Venkatesh K. All rights reserved</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
							</ul>
						</div>
					</footer>

			</div>

		<!-- Scripts -->
			<script src="/assets/js/jquery.min.js"></script>
			<script src="/assets/js/browser.min.js"></script>
			<script src="/assets/js/breakpoints.min.js"></script>
			<script src="/assets/js/util.js"></script>
			<script src="/assets/js/main.js"></script>

	</body>
</html>